### PR TITLE
Upload logs after rebuilding initrd in SLEM

### PR DESCRIPTION
**Pull request by mloviska**
SLEM5.1 image for s390x shows symptoms that point to rebuilding of
initrd on running system. In order to get more information it seems
useful to upload logs after rebuilding initrd as the system might end up
unbootable.

- VR http://kepler.suse.cz/tests/6813

### CODE REVIEW

File: tests/transactional/enable_selinux.pm [hunk 1]
```diff
@@ -30,6 +30,8 @@ sub run {
     # install and enable SELinux if not done by default
     if (script_run('test -d /sys/fs/selinux && test -e /etc/selinux/config') != 0) {
         assert_script_run('transactional-update setup-selinux');
+        upload_logs('/var/log/transactional-update.log');
+        shift->save_and_upload_log('rpm -qa','installed_pkgs.txt ');
```
[the remark this answers is not in the document]
Comment (jlausuch) [reply tests/transactional/enable_selinux.pm:34]: And not sure if we need to specify `{screenshot => 0}`

**Review by jlausuch — CHANGES_REQUESTED** [review]
I think there is a typo :)

File: tests/transactional/enable_selinux.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (mloviska) [reply tests/transactional/enable_selinux.pm:34]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/wicked/wlan.pm#L40

[the remark this answers is not in the document]
Comment (mloviska) [reply tests/transactional/enable_selinux.pm]: > And not sure if we need to specify `{screenshot => 0}`

No idea, I have checked `lib` files and it seems it is either omitted or set to 1.

[the remark this answers is not in the document]
Comment (jlausuch) [reply tests/transactional/enable_selinux.pm]: > https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/wicked/wlan.pm#L40

this link is pointing to different function, isn't it?

[the remark this answers is not in the document]
Comment (mloviska) [reply tests/transactional/enable_selinux.pm]: Sorry, I have misunderstood what is the problem. My point is that I don't need to do `my $self = shift`, instead I skip `$self` and use `shift->functiona_call`.

[the remark this answers is not in the document]
Comment (mloviska) [reply tests/transactional/enable_selinux.pm]: and the screenshot part, I understand it as optional parameter.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L88

**Review by jlausuch — APPROVED** [review]

**Pull request merged**